README.md

```diff
@@ -290,10 +290,12 @@ documentation:
   explains how you can define pipelines using YAML files.
```

Comment:
Can we also change the first two links here? I think we have equivalent pages on conduit.io. And if there are no substantial differences, we could even delete the docs from the repo.

Reply:
@lovromazgon i thought about doing that, but then I felt I was scope creeping a bit. I'll do a quick pass

Reply:
@lovromazgon done https://github.com/ConduitIO/conduit/pull/1732/commits/c524614621d425fdc6bd291813486af42e5a1012. Thought it needed more work.